AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginLazyLoadTests/LL11/AWSDataStoreLazyLoadPostComment8Tests.swift

```diff
@@ -68,12 +66,8 @@ final class AWSDataStoreLazyLoadPostComment8Tests: AWSDataStoreLazyLoadBaseTest
             XCTFail("Missing comments on post")
             return
         }
-        // This is a bit off, the post.identifier is the CPK while the associated field is just "postId",
-        // Loading the comments by the post identifier should be
-        // "query all comments where the predicate is field("@@postForeignKey") == "[postId]#[title]"
-        // List fetching is broken for this use case "uni directional has-many"
-        assertList(comments, state: .isNotLoaded(associatedId: post.identifier,
-                                                 associatedField: "postId"))
+        assertList(comments, state: .isNotLoaded(associatedIds: [post.postId, post.title],
+                                                 associatedFields: ["postId", "postTitle"]))
```

Comment:
Since the Post8+schema.swift codegenerated file has been updated to include targetNames, they are used and passed as "postId" and "postTitle", which are the fields on the Comment8 to build the filter on, when lazy loading